joomla/joomla-cms
# Merging database and warnings view to service view

See the title. This is meant to simplify the interface of Joomla and simply merges 2 views into one. We should do this in other places, too.
### Test instructions
- Check which messages you get in the warnings view and the database view before applying this patch.
- Apply the patch.
- Check that the same messages are displayed in the service view
- Force the situations that let messages show up in the warnings or database view and check that the same messages show up in the service view.
